.github/workflows/docs.yml

```diff
@@ -42,8 +42,8 @@ jobs:
           echo "flink_branch=${currentBranch}" >> ${GITHUB_ENV}
 
           if [ "${currentBranch}" = "master" ]; then
-            echo "flink_alias=release-1.18" >> ${GITHUB_ENV}
-          elif [ "${currentBranch}" = "release-1.17" ]; then
```

Comment:
We need to keep using `release-1.17` here as the latest stable version, and change it to 1.18 after we finally release the version.

Reply:
+1

Reply:
got it, thanks!

Reply:
I have addressed your suggestion. Please check it again, thanks!